AirCasting/Utils/CLLocationUtils.swift

```diff
@@ -0,0 +1,35 @@
+// Created by Lunar on 14/06/2021.
+//
+
+import CoreLocation
+
+extension CLLocationCoordinate2D: Equatable { }
+
+extension CLLocationCoordinate2D {
+    public init?(latitude: Double?, longitude: Double?) {
+        guard let lat = latitude, let long = longitude else { return nil }
+        self.init(latitude: .init(lat), longitude: .init(long))
+    }
+}
+
+public func ==(lhs: CLLocationCoordinate2D, rhs: CLLocationCoordinate2D) -> Bool {
+    return lhs.latitude == rhs.latitude && lhs.longitude == rhs.longitude
+}
+
+extension Optional where Wrapped == Double {
+    var locationDegrees: CLLocationDegrees {
```

Comment:
just use `double.flatMap(CLLocationDegrees.init)` rather than this. Not every double represents coordinate system and most of the times this would lead to confusion. I would also prefer to have the core location to be used in models and logic from the start so this conversion shouldn't be used almost at all.

Reply:
this was actually unused code, removed 👌